src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs

```diff
@@ -137,6 +138,12 @@ internal sealed class VisualStudioProject
                 (w, d) => w.OnAdditionalDocumentAdded(d),
                 (w, documentId) => w.OnAdditionalDocumentRemoved(documentId),
                 documentTextLoaderChangedAction: (w, d, loader) => w.OnAdditionalDocumentTextLoaderChanged(d, loader));
+
+            _analyzerConfigFiles = new BatchingDocumentCollection(this,
+                (s, d) => s.ContainsAnalyzerConfigDocument(d),
```

Comment:
Dumb question.  'AnalyzerConfig' is more abstract.  Do we ever expect this to not be a .editorconfig file though?  Should this just be ContainsEditorConfigDocument?  (same question for all the rest of the names).

Reply:
This is following the compiler API's naming convention that @agocke established Yes, there is a certain hedging that if we ever have a second format, this would cover that too.